Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/ucum/UcumServiceFactory.kt

```diff
@@ -0,0 +1,15 @@
+package org.cqframework.cql.cql2elm.ucum
+
+import java.util.*
+
+@Suppress("TooGenericExceptionThrown")
+object UcumServiceFactory {
+    fun load(): UcumService {
+        return ServiceLoader.load(UcumService::class.java).firstOrNull()
+            ?: throw RuntimeException(
```

Comment:
Shouldn't we be throwing something more specific, like IllegalStateException instead of RuntimeException in all cases, to signal the problem is with the state of the application, and not a user error?

Reply:
Fixed to use IllegalStateException